.github/workflows/auto-jdk-matrix.yml

```diff
@@ -0,0 +1,67 @@
+name: DataSketches-Memory Auto JDK Matrix Test & Install
+
+on:
+    pull_request:
+    push:
+    #    branches: [ master ]
```

Comment:
Dunno if you wanted this commented out? Certainly not a blocker to the PR either way, but ideally either uncomment or remove the line.

Reply:
Yeah, it needs to be uncommented before it goes into release branch.